lib/realtime/crowding.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Realtime.Crowding do
+  @type t :: %__MODULE__{
+          load: non_neg_integer(),
+          capacity: non_neg_integer(),
+          occupancy_status: String.t(),
+          occupancy_percentage: float()
```

Comment:
These should all be `| nil`, right, for the case where the route has crowding data but the vehicle doesn't?